### PR TITLE
Added remediation for aide_scan_notification rule.

### LINKED ISSUE #1873: needs remediation: xccdf_org.ssgproject.content_rule_aide_scan_notification

### PR DESCRIPTION
**Pull request by rsprudencio**
Fixes #1873

### CODE REVIEW

File: shared/templates/static/bash/aide_scan_notification.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+CRONTAB=/etc/crontab
+CRONDIRS='/etc/cron.d /etc/cron.daily /etc/cron.weekly /etc/cron.monthly'
+unset UPDATE_CRONTAB VARSPOOL
+
+if [ -f /var/spool/cron/root ]; then
+	VARSPOOL=/var/spool/cron/root
+fi
+
+if ! grep -qR '^.*\/usr\/sbin\/aide\s*\-\-check.*\|.*\/bin\/mail\s*-s\s*".*"\s*root@.*$' $CRONTAB $VARSPOOL $CRONDIRS; then
+	UPDATE_CRONTAB=1
```
Comment (mpreisler) [inline shared/templates/static/bash/aide_scan_notification.sh]: why use a separate variable, couldn't we just update it in the `if` body?

Comment (rsprudencio) [inline shared/templates/static/bash/aide_scan_notification.sh]: ahahaha yeah, this was for "legacy" reasons when I did the first version of the remediation, I'll change that :P

**Comment by rsprudencio** [comment]
@mpreisler updated, thanks

File: shared/templates/static/bash/aide_scan_notification.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# platform = Red Hat Enterprise Linux 7
+CRONTAB=/etc/crontab
+CRONDIRS='/etc/cron.d /etc/cron.daily /etc/cron.weekly /etc/cron.monthly'
+unset UPDATE_CRONTAB VARSPOOL
```
Comment (mpreisler) [inline shared/templates/static/bash/aide_scan_notification.sh]: UPDATE_CRONTAB is no longer used in the script, no need to unset it.

Comment (rsprudencio) [inline shared/templates/static/bash/aide_scan_notification.sh]: oops, removed

**Comment by mpreisler** [comment]
ACK

**Pull request merged**